locales/fr/LC_MESSAGES/base.po

```diff
@@ -22,87 +22,140 @@ msgstr ""
+"**Y a-t-il des frais pour utiliser Eco:** non"
 
 #: bot.py:74
 msgid "POINTS_TITLE"
-msgstr ""
+msgstr "Points Eco"
 
 #: bot.py:76
 msgid "POINTS_DESCRIPTION"
-msgstr ""
+msgstr "Les points Eco sont uniquement destinés à des fins de recherche"
 
 #: bot.py:82
 msgid "POINTS_FIELD1_NAME"
-msgstr ""
+msgstr "En savoir plus sur les points Eco"
 
 #: bot.py:83
 msgid "POINTS_FIELD1_VALUE"
 msgstr ""
+"**Pour vérifier combien de points vous avez:** entrez la commande `!points` dans la "
+"section #check-your-points\n"
+"**Comment envoyer des points Eco:** entrez la commande\n"
+"`!send @username number comment`\n"
+"**Comment gagner des points Eco:**\n"
+"1. Concourez pour des primes dans #content-bounties et faites des activités dans #points-bartering\n"
+"2. Participez à des discussions dans #general, #eco-nomics, et partout ailleurs\n"
+"3. Suivez les canaux, participez à des conversations et des concours\n"
+"4. Invitez des personnes au projet et ils peuvent vous remercier dans la section #invite-thank-yous. Seuls les utilisateurs avec un grand nombre "
+"d'invités obtiendront des points\n"
+"5. Créez des mèmes originaux (sans vulgarité) dans la section #meme-lab"
+"6. Soyez généralement utile au project et aidez la communauté à apprendre et à grandir ensemble!"
+
 
 #: bot.py:93
 msgid "ECO Pinned Messages"
-msgstr ""
+msgstr "Comment trouver des messages épinglés?"
 
 #: bot.py:103
 msgid "NAVIGATION_TITLE"
-msgstr ""
+msgstr "Navigation Eco"
 
 #: bot.py:105
 msgid "NAVIGATION_TITLE_DESCRIPTION"
-msgstr ""
+msgstr "Instructions pour naviguer sur le serveur"
 
 #: bot.py:111
 msgid "NAVIGATION_TITLE_FIELD1_NAME"
-msgstr ""
+msgstr "Canaux principaux"
 
 #: bot.py:112
 msgid "NAVIGATION_TITLE_FIELD1_VALUE"
 msgstr ""
+"#🏁start-here - passez de la Couche 0 à la Couche 1 ici\n"
+"#🚀general - discussion générale concernant le projet en anglais\n"
+"#📈crypto-chat - chat sur les cryptos, autres que Eco\n"
+"#🙏invite-thank-yous - ici vous pouvez remercier les gens qui vous ont invité au projet\n"
+"#🗺geo-tagging - vous pouvez épingler votre position sur Google Maps et voir d'où viennent d'autres "
+"membres de la communauté Eco\n"
+"#🏧check-your-points - regardez combien de points vous avez"
 
 #: bot.py:116
 msgid "NAVIGATION_TITLE_FIELD2_NAME"
-msgstr ""
+msgstr "Chats de contenu"
 
 #: bot.py:117
 msgid "NAVIGATION_TITLE_FIELD2_VALUE"
 msgstr ""
+"#🧪meme-lab - envoyez vos mèmes ici, uniquement ceux de bonne qualité\n"
+"#✅top memes - les mèmes les plus populaires sont sélectionnés et publiés ici par un bot\n"
+"#🖼eco-art - soumettez votre travail créatif à propos d'Eco - images, vidéos, et "
+"tout ce qui ne correspond pas aux mèmes et marketing\n"
+"#📹marketing-materials - soumettez des supports marketing Eco de qualité (brochures, "
+"affiches, merchandising, animations etc)\n"
+"#🏅content-bounties - l'information la plus récente sur les concours de primes est publiée ici par "
```

Comment:
I feel like `concours de primes` is wierd. But I don't have a better proposal in mind. Sorry

Reply:
propositions de `getordone` : 
- "concours recompensé" 👍 
- " concours avec gains"